### PR TITLE
Removed the link to trello and the code52 jabbr room. Added links to ref...

### PR DESCRIPTION
**Pull request by JakeGinnivan**
...actoring carnac blog posts

@shiftkey when you get a minute.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,17 +5,22 @@ A keyboard logging and presentation utility for presentations, screencasts, and
...
  * [Setting up Git for Windows and connecting to GitHub](http://help.github.com/win-set-up-git/)
  * [Forking a GitHub repository](http://help.github.com/fork-a-repo/)
  * [The simple guide to GIT guide](http://rogerdudler.github.com/git-guide/)
+ * [Raise an issue](https://github.com/Code52/carnac/issues) - simply create an issue if you have any ideas/find a bug
 
 Once you're familiar with Git and GitHub, clone the repository and run the ```.\build.cmd``` script to compile the code and run all the unit tests. You can use this script to test your changes quickly.
 
-### Discussing ideas 
+### Resources
+This blog series covers a series of refactorings which have recently happened in carnac to make better use of Rx.
```
Comment (shiftkey) [inline README.md]: Upper-case `Carnac`?

File: README.md [hunk 2]
```diff
@@ -5,17 +5,22 @@ A keyboard logging and presentation utility for presentations, screencasts, and
...
 
-### Getting started
+### Contributing
 
 **Getting started with Git and GitHub**
 
  * [Setting up Git for Windows and connecting to GitHub](http://help.github.com/win-set-up-git/)
  * [Forking a GitHub repository](http://help.github.com/fork-a-repo/)
  * [The simple guide to GIT guide](http://rogerdudler.github.com/git-guide/)
+ * [Raise an issue](https://github.com/Code52/carnac/issues) - simply create an issue if you have any ideas/find a bug
```
Comment (shiftkey) [inline README.md]: How about:

> `[Open an issue](https://github.com/Code52/carnac/issues)` if you encounter a bug or have a suggestion for improvements/features

File: README.md [hunk 3]
```diff
@@ -5,17 +5,22 @@ A keyboard logging and presentation utility for presentations, screencasts, and
...
 
 Once you're familiar with Git and GitHub, clone the repository and run the ```.\build.cmd``` script to compile the code and run all the unit tests. You can use this script to test your changes quickly.
 
-### Discussing ideas 
+### Resources
+This blog series covers a series of refactorings which have recently happened in carnac to make better use of Rx.
+If you are learning Rx and want to be shown through carnac's codebase then this blog series may help you.
```
Comment (shiftkey) [inline README.md]: :arrow_up:

**Comment by JakeGinnivan** [comment]
Fixed

**Pull request merged**